### DISCUSSION TITLE
【灌水】刚来绳网，怎么就60级了？？？🤔🤔🤔

### DISCUSSION THREAD

**Discussion by alonea13** [share121/inter-knot#1458] [其他]
![屏幕截图 2024-08-11 184414](https://github.com/user-attachments/assets/9cd711c7-61d0-4c66-81de-8f721730fd0d)

是不是出现_**bug**_了（雾）

**Comment by lllliufeng** [comment]
60级经验卡借我用一下（

**Comment by capacitor1** [comment]
![image](https://github.com/user-attachments/assets/b64770c0-a303-4393-8337-802a1ec56ea7)

doge

**Comment by xxtg666** [comment]
其实等级是你的github仓库数量（

**Comment by Mayf1ower** [comment]
貌似是你的github账号的public的仓库数量

**Comment by jjxya** [comment]
肯定是F12
